cfme/tests/cloud/test_providers.py

```diff
@@ -502,7 +503,10 @@ def test_azure_subscription_required(request, provider):
 @pytest.mark.tier(2)
 @test_requirements.azure
 @pytest.mark.provider([AzureProvider], scope="function", override=True, selector=ONE)
-def test_azure_multiple_subscription(appliance, request, soft_assert):
+@pytest.mark.provider([AzureProvider], fixture_name="second_provider", selector=SECOND)
```

Comment:
@lina-nikiforova, I'm curios how we can test this at all since we have only one azure provider in cfme_data ?  As far as I understand two subscriptions implies two azure providers.  
I guess we need to add one more azure provider with another subscription to test this.

@mshriver, @john-dupuy, what's your opinion ?

Reply:
@izapolsk don't we have 2? I see `azure` and `azure_msdn` in the yamls

Reply:
@john-dupuy, you're right my bad, I incorrectly interpreted search results.